src/porting.cpp

```diff
@@ -627,6 +627,36 @@ bool setXorgWindowIcon(IrrlichtDevice *device)
 		setXorgWindowIconFromPath(device,
 			path_share + "/misc/" PROJECT_NAME "-xorg-icon-128.png");
 #	endif
+#elif defined(_WIN32)
+	const video::SExposedVideoData exposedData = device->getVideoDriver()->getExposedVideoData();
+	HWND hWnd; // Window handle
+
+	switch (device->getVideoDriver()->getDriverType()) {
+	case video::EDT_DIRECT3D8:
+		hWnd = reinterpret_cast<HWND>(exposedData.D3D8.HWnd);
+		break;
+	case video::EDT_DIRECT3D9:
+		hWnd = reinterpret_cast<HWND>(exposedData.D3D9.HWnd);
+		break;
+	case video::EDT_OPENGL:
+		hWnd = reinterpret_cast<HWND>(exposedData.OpenGLWin32.HWnd);
+		break;
+	default:
```

Comment:
No. Thats not available. http://irrlicht.sourceforge.net/docu/structirr_1_1video_1_1_s_exposed_video_data.html